RubeesEat/Pages/SplitBill.cshtml

```diff
@@ -1,28 +1,43 @@
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/site.css" />
+</head>
 
-    <label for="billDescription">Description:</label>
-    <input id="billDescription" t-id="billDescription" type="text" name="billDescription" required="required"></input><br>
+<div class="container mt-3">
+    <h1 class="text-center">Split bill:</h1>
+    <form id="splitBillForm" method="post" action="/api/bills" class="mx-auto mt-4" style="max-width: 600px;">
+        <div class="mb-3">
+            <label for="billDescription">Description:</label>
+            <input class="form-control" style="max-width: 900px;" id="billDescription" t-id="billDescription" type="text" name="billDescription" required="required"></input><br>
+        </div>
 
-    <label for="billAmount">Total Price:</label>
-    <input id="billAmount" t-id="billAmount" type="number" step=".01" name="billAmount" required="required"></input><br>
+        <div class="mb-3">
+            <label for="billAmount">Total Price:</label>
+            <input class="form-control" id="billAmount" t-id="billAmount" type="number" step=".01" name="billAmount" required="required"></input><br>
+        </div>
 
-    <div id="addedPeople" t-id="addedPeople"></div>
-    <div id="addPerson">
-        <label for="billPeople">Add Person:</label>
-        <select name="billPeople" id="billPeople" t-id="billPeople">
-            @foreach (Person person in PersonRepository.GetAll())
-            {
-                <option value="@person.FirstName @person.LastName" data-id="@person.Id">@person.FirstName @person.LastName </option>
-            }
-        </select>
-        <input t-action="addPerson" t-click-behavior="Click" type="button" onclick="AddPerson()" value="Add Person"></input><br/>
-    </div>
+        <div id="addedPeople" t-id="addedPeople"></div>
+        <div class="mb-3" id="addPerson">
+            <label for="billPeople">Add Person:</label>
+            <div class="d-flex">
 
-    <button t-action="splitBill" t-click-behavior="Click" type="button" onclick="validate()">Split the bill</button><br>
-    <text id="placeForErrorMessage" t-id="placeForErrorMessage" style="color: red;"></text>
-</form>
+                <select class="form-select" name="billPeople" id="billPeople" t-id="billPeople">
+                    @foreach (Person person in PersonRepository.GetAll())
+                    {
+                        <option value="@person.FirstName @person.LastName" data-id="@person.Id">@person.FirstName @person.LastName </option>
+                    }
+                </select>
+                <input t-action="addPerson" t-click-behavior="Click" type="button" onclick="AddPerson()" value="Add Person"></input><br/>
+            </div>
+        </div>
+
+        <button t-action="splitBill" t-click-behavior="Click" type="button" onclick="validate()">Split the bill</button><br>
```

Comment:
This button and all the buttons added via JS need a button class to apply the bootstrap styling. We can also add a primary color to the "OK" button.